### ISSUE TITLE
In dxtbx DataBlockTemplateImporter allow for option to work around missing images

### ISSUE THREAD

**Issue by graeme-winter** [cctbx/cctbx_project#255]
i.e. if I have a sequence of images 

```
th_8_2_0310.cbf
th_8_2_0311.cbf
th_8_2_0312.cbf
th_8_2_0313.cbf
th_8_2_0314.cbf
th_8_2_0316.cbf
th_8_2_0317.cbf
th_8_2_0318.cbf
th_8_2_0319.cbf
```

let the caller have the option of dealing with this. Use case is where caller will subsequently be trimming the sweep anyway. Yes, could work around this by calculating a full list of images and trimming to what is wanted, but seems like a useful option.

**Comment by graeme-winter** [comment]
Approach: **kwargs is available as a mechanism to pass this down - use ```allow_incomplete_sweeps```

**Comment by ndevenish** [comment]
Slightly worried that this is making the behaviour between template/non-template even further apart. I guess this should also handle `allow_multiple_sweeps` and normal importing should `allow_incomplete_sweeps`?

**Comment by graeme-winter** [comment]
That's a fair comment. I'm not sure what is the right answer on this one.

**Comment by graeme-winter** [comment]
@ndevenish do you have another ticket open about "making the behaviour between template/non-template even further apart" - if no, probably worth opening one, so I can close this?

**Comment by ndevenish** [comment]
No, it's more of a long term interface worry - feel free to close this
